doc/release/2021-08-releases.rst

```diff
@@ -49,28 +52,140 @@ Core
-    ``box.info.synchro`` interface (:tarantool-release:`2.7.3`, :tarantool-issue:`5191`).
+    ``box.info.synchro`` interface
+    (:tarantool-release:`2.7.3`, :tarantool-release:`2.10.0-beta1`, :tarantool-issue:`5191`).
+
+-   Field type UUID is now part of field type ``SCALAR``
+    (:tarantool-release:`2.10.0-beta1`, :tarantool-issue:`6042`).
+
+-   Field type ``UUID`` is now available in SQL, and a new UUID can be generated
+    using the new SQL built-in function ``UUID()``
+    (:tarantool-release:`2.10.0-beta1`, :tarantool-issue:`5886`).
+
+-   **[Breaking change]** The ``timeout()`` method of the ``net.box`` connection,
+    marked deprecated more than four years ago (in ``1.7.4``), has been dropped.
+    It negatively affected the performance of hot ``net.box` methods, such as ``call()`` and ``select()``
+    if those were called without specifying a timeout
+    (:tarantool-release:`2.10.0-beta1`, :tarantool-issue:`6242`).
+
+-   Improved ``net.box`` performance by up to 70% by rewriting hot code paths in C
+    (:tarantool-release:`2.10.0-beta1`, :tarantool-issue:`6241`).
+
+-   Introduce compact tuples that allow saving 4 bytes per tuple in case of small userdata
+    (:tarantool-release:`2.10.0-beta1`, :tarantool-issue:`5385`)
+
+-   Now streams and interactive transactions over streams are implemented in iproto.
+    Stream is associated with its ID, which is unique within one connection.
+    All requests with the same, not zero stream ID, belong to the same stream.
+    All requests in the stream are processed synchronously.
+    The execution of the next request will not start until the previous one is completed.
+    If a request has zero stream ID, it does not belong to a stream and is processed in the old way.
+
+    In ``net.box`, a stream is an object above the connection that has the same methods
+    but allows to execute requests sequentially. ID is generated on the client side automatically.
+    If a user writes his own connector and wants to use streams, he must transmit ``stream_id`` over the iproto protocol.
+
+    The primary purpose of streams is transactions via iproto.
+    Each stream can start its transaction, so they allow multiplexing several transactions over one connection.
+    There are multiple ways to begin, commit and rollback a transaction.
+    One can do that using appropriate stream methods, using ``call`` or ``eval`` methods,
+    or the ``execute`` method with SQL transaction syntax. User can mix these methods.
+    For example, start a transaction using ``stream:begin()``,
+    and commit transaction using ``stream:call('box.commit')`` or ``stream:execute('COMMIT')``.
+    If any request fails during the transaction, it will not affect the other requests in the transaction.
+    If a disconnect occurs when there is some active transaction in the stream,
+    this transaction will be rollbacked if it does not have time to commit before this moment.
```

Comment:
```suggestion
    One can do that using the appropriate stream methods, ``call``, ``eval``,
    or ``execute`` with the SQL transaction syntax. Users can mix these methods.
    For example, one might start a transaction using ``stream:begin()``
    and commit it with ``stream:call('box.commit')`` or ``stream:execute('COMMIT')``.
    If any request fails during the transaction, it will not affect the other requests in the transaction.
    If a disconnect occurs while there is an active transaction in the stream,
    that transaction will be rolled back if it hasn't been committed before the connection failure.
```